app/src/components/submissionPage/SnapSubmissionStyles.ts

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components";
+import { MentionsInput } from "react-mentions";
+
+/* A list of styled components for SubmissionTextBox.tsx */
+
+export const ElleImg = styled.img`
+    width: 100%;
+    max-height: 250px;
+`;
+
+export const LeftSideOfScreen = styled.div`
+    height: 430px;
+    background-color: #7040d6;
```

Comment:
Look for opportunities to commonise some of the styles. The `height` and `background-color` or `LeftSideOfScreen` should probably always match `RightSideOfScreen`, for example.